.gitignore

```diff
@@ -18,6 +18,7 @@
 /Library/PinnedKegs
 /Library/PinnedTaps
 /Library/Taps
+/Library/lock.d
```

Comment:
Agreed. That should probably be a place where we're also unlikely to be affected by permission issues and sadly `/usr/local/Library` doesn't always fall into that category.